### ISSUE TITLE
"In This Section" expandable doesn't open on non-wagtail pages

### ISSUE THREAD

**Issue by KimberlyMunoz** [cfpb/cfgov-refresh#1801]
Short description explaining the high-level reason for the new issue. This is the opposite problem of #1673

## Current behavior

The "In this section" links do not expand or show the expanded state.

## Expected behavior

- The "In this section" should either expand or be expanded so people can navigate.

## Steps to replicate behavior

1. Visit http://flapjack.demo.cfpb.gov/about-us/the-bureau/ on mobile

## Screenshots
![image](https://cloud.githubusercontent.com/assets/1860176/14252758/b505baac-fa56-11e5-92fe-e797dbef66e1.png)

**Comment by jimmynotjim** [comment]
This is working locally after #1802 merged, can we close it?

**Comment by KimberlyMunoz** [comment]
Aren't we still doing the "review on next refresh push" thing? I'm fine closing it, but let's kill that tag or clarify when to use it next retro!
